public/js/actions/sources.js

```diff
@@ -52,6 +52,30 @@ function newSource(source) {
   };
 }
 
+function loadSourceMap(generatedSource) {
+  return ({ dispatch, getState }) => {
+    let sourceMap = getSourceMap(getState(), generatedSource.id);
+    if (sourceMap) {
+      return {};
```

Comment:
oh, really? Interesting, you can see in `selectSource` we just `return`, so not sure what was throwing that